### ISSUE TITLE
Update compression for current object store - move to Zarr

### ISSUE THREAD

**Issue by gareth-j** [openghg/openghg#561]
This might be low priority but if we have a large object store with the current NetCDF write system, could we include the functionality to read the current data, compress it and replace the existing NetCDFs with Zarr stores?

**Comment by gareth-j** [comment]
- For now keep the storage and post-processing things separate.
- Can we convert current object stores to use the new compression methods?

**Comment by rt17603** [comment]
Can also use of `pytest-memray` to profile memory usage and include within the tests (see also https://github.com/openghg/openghg/issues/456 ).

**Comment by rt17603** [comment]
> # TODO
> ...

@gareth-j How much of this TODO is done / still needed? And is there anything which needs to be split out into separate issues for focus on for this?

**Comment by rt17603** [comment]
Linked to #803 "Move to zarr storage" - once this PR is merged into the `new_object_store` branch, merging #831 will close this issue
